Breeze.UI/src/app/wallet/tumblebit/tumblebit.service.ts

```diff
@@ -24,6 +24,7 @@ export class TumblebitService {
   // Might make sense to populate tumblerParams here because services are singletons
 
   connectToTumbler(operation: 'connect' | 'changeserver'): Observable<any> {
+    console.info('connectToTumbler');

```

Comment:
We should really use logger service instead of console.info as it won’t do logs in prod release. https://github.com/BreezeHub/BreezeProject/blob/master/Breeze.UI/src/app/shared/services/logger.service.ts

Reply:
Sure, OK